### ISSUE TITLE
Error TS2355, TS1183

### ISSUE THREAD

**Issue by johan3040** [KittyGiraudel/react-a11y-dialog#148]
Hello! 

First of all, thank you so much for this easy to use lib.

I am, however, having issues with typescript errors then using the hook `useA11yDialog`.

In index.d.ts
![image](https://user-images.githubusercontent.com/37670348/160361666-109ef632-dcf7-4718-9ede-a57c19cd5be5.png)

`node_modules/react-a11y-dialog/index.d.ts:52:4 - error TS2355: A function whose declared type is neither 'void' nor 'any' must return a value.`
`node_modules/react-a11y-dialog/index.d.ts:52:43 - error TS1183: An implementation cannot be declared in ambient contexts.`

Is this something that is in progress to be fixed?

**Comment by johan3040** [comment]
Possible fix would be to remove the function body:

`export function useA11yDialog(
  props: A11yDialogProps
): [A11yDialogInstance, A11yDialogConfig]`

**Comment by KittyGiraudel** [comment]
Hello, and thank you for opening an issue! I know nothing about TypeScript, so I took the liberty to [ask for help on Twitter](https://twitter.com/KittyGiraudel/status/1508382122737905669?s=20&t=NHuNkO_Sn6Kd_5yF94jLYw). If you believe your suggestion would fix the problem, feel free to issue a PR and I’ll publish a fix as soon as it’s merged. :)

**Comment by johan3040** [comment]
Thank you for such a fast response 😊. I have opened up a PR resolving the issue here https://github.com/KittyGiraudel/react-a11y-dialog/pull/151

**Comment by KittyGiraudel** [comment]
Fixed in https://github.com/KittyGiraudel/react-a11y-dialog/releases/tag/6.1.5. 😊  Thank you for the fix!
